### ISSUE TITLE
calibrate.recalibrate() debug

### ISSUE THREAD

**Issue by bplimley** [bearing/radwatch-analysis#53]
Pull request #44 seems to have introduced a bug into the `recalibrate()` function in `calibrate.py`. `recalibrate()` is doing something with the `_recal` files and so I think it conflicts with `acquire_files()` now.

**Comment by laurac8r** [comment]
Hi Brian,

Can you provide more specifics on the issue? It doesn't seem to be occurring from my end. If anything, `make_table` does throw an error about the `RadWatch_Samples.csv` metadata not matching the SPE files list, but that is a simple fix in the .csv file. My first guess would be removing `_recal` in `sample_measurements` in `acquire_files()` has some cross-interference with `recalibrate()`.

I can look at the code logic closely tomorrow, in any case.

**Comment by bplimley** [comment]
@yarocoder can you `run calibrate.py` successfully?

I don't think we want to change `acquire_files()` at all because I like it the way you fixed it now. But I have to look at what `recalibrate()` was trying to do with `_recal`. I'll have to look at it closer tomorrow or Monday.

**Comment by laurac8r** [comment]
Sounds good!

**Comment by laurac8r** [comment]
Hi Brian,

The answer is apparent: `acquire_files()` removes the original versions of the recal files, not the recal files themselves, like you intended and was resolved in the `calibrate.py` script. `recalibrate()` takes the `sample_measurements` output from `acquire_files()` and _only accepts_ non-recal list elements and creates `cal_offsets` with them. Inherently, `sample_measurements` and `cal_offsets` will not be the same length and we get the error observed. I've fixed this and running `calibrate.py` has no error now and I'll upload the script with a simple fix included that doesn't mess with the functionality of any of the functions involved while resolving the error.

For the CSV meta data error when running `Gamma_Analysis.py`, the primary issue is the inclusion of artificial recal files we've added, as well as the removal of `UCB018_Soil_Sample010_2.Spe` from 'sample_names' while `acquire_files()` is appending spectrum files to `sample_measurements`, from which `sample_names` is created. We need to remove `UCB018_Soil_Sample010_2.Spe` from the CSV file, to skip it during the parsing of the CSV file, or count one less spectrum file in the `make_table()` function in `Gamma_Analysis.py`. I suggest the latter. Brian, I'll wait for your input before proceeding.
